docs/source/cpp/api/utilities.rst

```diff
@@ -26,6 +26,10 @@ Decimal Numbers
    :project: arrow_cpp
    :members:
 
+.. doxygenclass:: arrow::Decimal256
+   :project: arrow_cpp
```

Comment:
Seems we are a little inconsistent on whether we specify [`:project:`](https://breathe.readthedocs.io/en/v2.0.0/directives.html#directives) or not.  `arrow_cpp` is the [default](https://github.com/apache/arrow/blob/master/docs/source/conf.py#L88) so this should be redundant.

Reply:
Hmm, yes, I can try to remove all instances from this file.